ThingIFSDK/ThingIFSDK/AbstractThing.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  AbstructThing.swift
+//  ThingIFSDK
+//
+import Foundation
+
+/** Represents Target */
+public class AbstractThing : NSObject, TargetThing {
+    public let typedID: TypedID
+    public let accessToken: String?
```

Comment:
There is not `accessToken` in Android SDK. Can you check whether it is necessary?